### DISCUSSION TITLE
Dragging Pinned Tabs (or possible future essential tabs) to separate windows, while keeping the tabs pinned.

### DISCUSSION THREAD

**Discussion by ironcloudy** [zen-browser/desktop#2468] [Ideas]
Something small but I think very useful I like about Arc pinned or fav tabs is that when dragging them out of the current window, they don't get removed from the pin or fav section; dragging them out simply create a copy of the pinned page in a new window. This is especially useful for folks using large monitor or multi monitors.

Perhaps Zen can consider that as well.

**Comment by BenjaminAlbertR** [comment]
Drag pinned tabs in zen browser.
I really want this. The times I want to make a new window, I usually want to use the pinned section. For me, it's like a dictionary or something. And instead I need to create a new tab, drag'n drop and then select the pin. Would be a good quality of life improvement. :)
